src/utils/Logger.ts

```diff
@@ -1,22 +1,16 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 export class Logger {
-    static info(tag: string, message: string, context?: any) {
-        if (process.env.REACT_APP_DEVELOPER_MODE !== 'true') {
+    static debug(tag: string, message: string, context?: unknown) {
+        if (process.env.REACT_APP_DEBUG !== 'true') {
             return;
         }
-        console.log(`[${tag}] : ${message}`, { context });
+        console.debug(`[${tag}] : ${message}`, { context });
     }
 
-    static debug(tag: string, message: string, context?: any) {
-        if (process.env.REACT_APP_DEVELOPER_MODE !== 'true') {
```

Comment:
Why are you removing these ifs

We dont want any logs in production

I dont know why are you even making any changes in this file in scope od this task

Reply:
we don't want errors? yeah ok whatever